apps/src/music/blockly/setup.ts

```diff
@@ -54,7 +54,7 @@ export function setUpBlocklyForMusicLab() {
     Blockly.JavaScript[blockType] = blockConfig.generator;
   }
 
-  if (getBlockMode() !== BlockMode.ADVANCED) {
+  if (blockMode !== BlockMode.ADVANCED) {
```

Comment:
This is where we may have to split things up as mentioned above, since this code is only meant to run once per page load.